classes/Tutor.php

```diff
@@ -837,13 +856,134 @@ public static function create_database() {
 			PRIMARY KEY (withdraw_id)
 		) $charset_collate;";
 
+		$orders_table = "CREATE TABLE {$wpdb->prefix}tutor_orders (
+			id BIGINT(20) UNSIGNED AUTO_INCREMENT,
+			payment_id BIGINT(20) UNSIGNED COMMENT 'Unique payment id from payment gateway',
+			user_id BIGINT(20) UNSIGNED NOT NULL,
+			status ENUM('pending', 'completed', 'cancelled', 'refunded', 'partially-refunded') NOT NULL,
+			sub_total_price DECIMAL(13, 2) NOT NULL, -- price calculation based on course sale price
```

Comment:
`subtotal` is a single word.
```suggestion
			subtotal_price DECIMAL(13, 2) NOT NULL, -- price calculation based on course sale price
```

Reply:
updated